hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestRegionServerReportForDuty.java

```diff
@@ -223,6 +226,41 @@ public void testReportForDutyWithMasterChange() throws Exception {
 
   }
 
+  /**
+   * Tests region sever reportForDuty with manual environment edge
+   */
+  @Test(timeout = 60000)
```

Comment:
timeout is not required here. This belongs to `LargeTests`.

Reply:
Makes sense, I will remove the timeout 

I have seen TestClockSkewDetection, the tests basically mocks a server and just tests the registration between RS and master. The current test in TestRegionServerReportForDuty also checks if the master comes online and finishes the initialization without any issue when a manual edge is inserted which is where i could see the gap in RSProcedureDispatcher and made the change. Please let me know if you think otherwise

Reply:
The change basically allows you to set configuration parameter for the retry of RPC to go through successfully in the case where is some custom delegate for the environment edge. We can use this even for any tests in future where there is manual environment edge injected. Let me know your thoughts on this

Reply:
Yeah, this looks good 👍